### DISCUSSION TITLE
Allow making production builds higher priority than preview ones

### DISCUSSION THREAD

**Discussion by osdiab** [vercel/vercel#5485] [Ideas]
At my company's current concurrency levels we often have builds queued, which for the most part is fine; but I wish I could tell Vercel to have production deploys be higher priority and jump the queue ahead of Preview ones, even potentially temporarily pausing or cancelling and rescheduling an in-progress preview build. Not sure if that's happening or not already in some form but it would make deploy timing nicer for my company.

**Comment by osdiab** [comment]
Bummer no comment on this... Guess it does go against Vercel's pricing scheme :P

**Comment by goleary** [comment]
Looks like this is available as an enterprise feature:
https://vercel.com/changelog/prioritize-production-deployments-to-build-before-any-queued-preview

**Reply by osdiab** [reply]
lol, @leerob another reason i just use [the Vercel CLI/unofficial github action](https://github.com/amondnet/vercel-action/issues/103) instead of Vercel's build system.
